.github/workflows/pr-checks.yml

```diff
@@ -82,6 +82,8 @@ jobs:
           template: "@/contrib/sarif.tpl"
           output: "trivy-results.sarif"
           severity: "CRITICAL,HIGH"
+        env:
+          TRIVY_DB_REPOSITORY: public.ecr.aws/aquasecurity/trivy-db:2
```

Comment:
```suggestion
          # use an alternative trivvy db to avoid rate limits
          TRIVY_DB_REPOSITORY: public.ecr.aws/aquasecurity/trivy-db:2
```